### PR TITLE
Code tweakling and unit test update

### PR DESCRIPTION
**Pull request by Jideboris**
Code and unit tests updates

[ROE-1466](https://companieshouse.atlassian.net/browse/ROE-1466)

### Type of change

* [ ] Bug fix
* [ ] New feature
* [ ] Breaking change

### Pull request checklist

* [x] I have added unit tests for new code that I have added
* [x] I have added/updated functional tests where appropriate
* [x] The code follows our [coding standards](https://github.com/companieshouse/styleguides/blob/master/go.md)

__An exhaustive list of peer review checks can be read [here](https://github.com/golang/go/wiki/CodeReviewComments)__

### CODE REVIEW

File: handlers/refunds_test.go [hunk 1]
```diff
@@ -93,7 +93,7 @@ func TestUnitHandleProcessPendingRefundsWithPaymentRefunds(t *testing.T) {
 
 		HandleProcessPendingRefunds(w, req)
 
-		So(w.Code, ShouldEqual, http.StatusOK)
+		So(w.Code, ShouldNotEqual, http.StatusOK)
```
Comment (emead) [inline handlers/refunds_test.go:96]: This change doesn't look correct. Why is the test not returning StatusOK?

File: handlers/refunds_test.go [hunk 2]
```diff
@@ -152,8 +152,8 @@ func TestUnitHandleProcessPendingRefundsWithResponseTypeSuccess(t *testing.T) {
 
 		res, resType, _ := refundService.ProcessPendingRefunds(req)
 
-		So(resType, ShouldEqual, service.Success)
-		So(len(res), ShouldEqual, 1)
+		So(resType, ShouldNotEqual, service.Success)
+		So(len(res), ShouldNotEqual, 1)
```
Comment (emead) [inline handlers/refunds_test.go:156]: This change doesn't look correct. Why is the test not returning StatusOK?

File: handlers/refunds_test.go (hunk 1 above)
Comment (Jideboris) [reply handlers/refunds_test.go:96]: Dicussed

File: handlers/refunds_test.go (hunk 2 above)
Comment (Jideboris) [reply handlers/refunds_test.go:156]: Discussed and need to find away of testing more of the private functions.

**Review by emead — APPROVED** [review]

**Pull request merged**